ios/FluentUI/Core/Colors.swift

```diff
@@ -288,14 +287,14 @@ public final class Colors: NSObject {
     }
 
     public struct Badge {
-        public static var background: UIColor = primaryTint40
```

Comment:
I like this change a lot as is - making these computed should be a huge help as we figure out how to make colors truly dynamic for the case of multiple fluent components with different themes active at the same time (multi-window).

Getting greedy, is this also a chance to start pushing some of these out of Colors.swift into the files they actually affect? Would help a lot with clarity of code and understanding impact of changes, but also could help with bundle size if we go down a path where a client can handpick the controls they want and not get all of them

Reply:
That feels like a separate change to me logically, even if it's something we choose to do next.

Reply:
i understand the desire and completely agree with your direction. but i don't think it is something i should do in _this_ PR

Reply:
Yup, makes sense